src/privacy-policy.md.njk

```diff
@@ -5,52 +5,105 @@ layout: layout-single-page-prose.njk
+The [GOV.UK Design System](https://design-system.service.gov.uk/) brings together the latest research, design and development from across government to make sure it’s representative and relevant for its users. 
 
-We collect:
+To add to and improve the Design System, the team reviews proposals and works with contributors to publish new entries and improve existing ones.  
 
-- questions, queries or feedback you leave
-- your email address, if you contact us
+The GOV.UK Design System is provided by the [Government Digital Service (GDS)](https://www.gov.uk/government/organisations/government-digital-service/about) which is part of the Cabinet Office. The data controller for GDS is the Cabinet Office - a data controller determines how and why personal data can be processed. 
 
-This data can be viewed by authorised people in the [Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) (GDS) and our suppliers, to:
+[Read the Cabinet Office’s entry in the Data Protection Public Register for more information](https://ico.org.uk/ESDWebPages/Entry/Z7414053).
 
-- gather feedback to improve our services
-- respond to any feedback you send us, if you’ve asked us to.
+## Why we need your data
 
-## Where your data is stored
+For a number of the activities that we undertake to complete our function, we need to process personal data. The purposes are to manage and coordinate the [GOV.UK Design System](https://design-system.service.gov.uk/) and contributions from the community. 
 
-We store your data on secure servers in the [European Economic Area](https://www.gov.uk/eu-eea) (EEA).
+This includes:
+- contribution process, including working with you on contributions, where you’ve proposed to add or improve part of the Design System
+- support, including both the provision of support and responses to user enquiries
+- feedback, including gathering it to improve our services, and responding to it, if you have asked us to
+- community, which includes telling you about updates to the Design System, and how to contribute to the Design System and collaborate with the community, if you’ve asked us to
 
-## Keeping your data secure
+## What data we collect from you
 
-Sending information over the internet is generally not completely secure, and we cannot guarantee the security of your data while it’s in transit.
+We collect certain information and data about you when you use the [GOV.UK Design System](https://design-system.service.gov.uk/). 
 
-**Any data you send is at your own risk**.
+We collect your:
+- name
+- business contact details
+- user profile on collaboration tools and platforms
 
-We have procedures and security features in place to keep your data secure once we receive it.
+If you sign up to our mailing list, we'll collect your:
+- name
+- email address and business contact details
 
-## Disclosing your information
+## Our Legal Basis for processing your data
```

Comment:
No reason apparently — I assumed it was the name of a specific rule or law but I can't find any evidence of this.
```suggestion
## Our legal basis for processing your data
```